posthog/queries/trends/formula.py

```diff
@@ -1,21 +1,22 @@
 import math
 from itertools import accumulate
+from string import ascii_uppercase
 from typing import Any, Dict, List
 
-from sentry_sdk import capture_exception, push_scope
+from sentry_sdk import push_scope
 
 from posthog.clickhouse.kafka_engine import trim_quotes_expr
 from posthog.client import sync_execute
 from posthog.constants import NON_TIME_SERIES_DISPLAY_TYPES, TRENDS_CUMULATIVE
 from posthog.models.filters.filter import Filter
 from posthog.models.team import Team
 from posthog.queries.breakdown_props import get_breakdown_cohort_name
-from posthog.queries.trends.util import parse_response
+from posthog.queries.trends.util import ensure_value_is_json_serializable, parse_response
 
 
 class TrendsFormula:
     def _run_formula_query(self, filter: Filter, team: Team):
-        letters = [chr(65 + i) for i in range(0, len(filter.entities))]
+        letters = [ascii_uppercase[i] for i in range(0, len(filter.entities))]
```

Comment:
```suggestion
        letters = [char for char, i in zip(ascii_uppercase, filter.entities)]
```
If you want to get fancy ;)

Reply:
This is nice. Though I'm gonna stick to the smaller flyby change in this one 😄